packages/patternfly-4/react-core/src/components/AboutModal/AboutModal.tsx

```diff
@@ -51,14 +56,26 @@ export class AboutModal extends React.Component<AboutModalProps> {
       // tslint:disable-next-line:no-console
       console.error('AboutModal:', 'brandImageAlt is required when a brandImageSrc is specified');
     }
+    this.state = {
+      container: undefined
+    };
   }
 
-  private handleEscKeyClick = (event: KeyboardEvent) => {
+  handleEscKeyClick = (event: KeyboardEvent) => {
     if (event.keyCode === KEY_CODES.ESCAPE_KEY && this.props.isOpen) {
       this.props.onClose();
     }
   };
 
+  toggleSiblingsFromScreenReaders = (hide: boolean) => {
+    const bodyChildren = document.body.children;
+    for (const child of Array.from(bodyChildren)) {
+      if (child !== this.state.container) {
```

Comment:
`this.state.container` will always be undefined?

Reply:
I see what you mean ... My understanding of this function is that as long as a child of `bodyChildren` is not undefined, it is assigned an `ariaHidden` value of either true or false. Is that correct?